engine/src/CalciteInterpreter.cpp

```diff
@@ -605,8 +631,9 @@ blazing_frame evaluate_split_query(std::vector<ral::io::data_loader> input_loade
 		// process yourself and return
 
 		if(is_scan(query[0])) {
-			blazing_frame scan_frame;
-			std::vector<gdf_column_cpp> input_table;
+			std::vector< std::unique_ptr<cudf::column> > scan_frame_columns;
+			std::vector< std::unique_ptr<cudf::column> > input_table;
+			std::vector<std::string> col_names;
```

Comment:
use a BlazingTable dont make a seperate container for col_names

Reply:
yep fixed